src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/InstancesController.cs

```diff
@@ -283,11 +279,11 @@ public class InstancesController : ControllerBase
             }
             catch (PlatformHttpException e)
             {
-                HandlePlatformHttpException(e, $"Instatiation of data elements failed for instance {instance.Id} for party {instanceTemplate.InstanceOwner?.PartyId}");
+                HandlePlatformHttpException(e, $"instantiation of data elements failed for instance {instance.Id} for party {instanceTemplate.InstanceOwner?.PartyId}");
```

Comment:
captial I

Reply:
I rewrote exception handling as you suggested , removing the line entirely. Notice how it doesn't have a return statement. Must have been a bug, right?